module1.js

```diff
@@ -64,9 +75,8 @@ function question4(array) {
 */
 
 function question5(number1, number2) {
-  // TODO:
+  return number1 * number2;
```

Comment:
Great!
This is your clue to how to write out `question1` and `question2` implementations on 1 line :)